### ISSUE TITLE
[Flaky CI] [test_id:2555] fail creating import dv: invalid qcow large size

### ISSUE THREAD

**Issue by oshoval** [kubevirt/containerized-data-importer#1358]
/kind bug
/triage build-officer

**What happened**:
Test is flaky 
https://prow.apps.ovirt.org/view/gcs/kubevirt-prow/pr-logs/pull/kubevirt_containerized-data-importer/1353/pull-containerized-data-importer-e2e-k8s-1.17-ceph/1302531371046014977

```
Timed out after 270.000s.
Expected
    <bool>: false
to be true
```

**What you expected to happen**:
Test should be stable

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- CDI version (use `kubectl get deployments cdi-deployment -o yaml`):
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- Install tools:
- Others:

**Comment by awels** [comment]
Fixed by #1363

**Comment by tomob** [comment]
This test keeps failing:

- https://cnv-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/test-cdi-ocp-4.6-cnv-2.5-prod/103/testReport/(root)/Tests%20Suite/_vendor_cnv_qe_redhat_com__level_component_DataVolume_tests_Verify_DataVolume_should__rfe_id_1120__crit_high__posneg_negative__test_id_2555_fail_creating_import_dv__invalid_qcow_large_size/
- https://cnv-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/test-cdi-ocp-4.6-cnv-2.5-prod/120/testReport/(root)/Tests%20Suite/_vendor_cnv_qe_redhat_com__level_component_DataVolume_tests_Verify_DataVolume_should__rfe_id_1120__crit_high__posneg_negative__test_id_2555_fail_creating_import_dv__invalid_qcow_large_size/
